osmosis-types/gamm/pool-models/stableswap/pool.go

```diff
@@ -41,7 +42,7 @@ func NewStableswapPool(poolId uint64, stableswapPoolParams PoolParams, initialLi
 }
 
 func (p Pool) GetAddress() sdk.AccAddress {
-	addr, err := sdk.AccAddressFromBech32(p.Address)
+	_, addr, err := bech32.DecodeAndConvert(p.Address)
```

Comment:
same as prev why not use `func AccAddressFromBech32(address string, checkHRP string)`

Reply:
this still seems not using utils one

Reply:
fixed.